Orange/widgets/regression/owrandomforestregression.py

```diff
@@ -1,35 +1,24 @@
-from PyQt4 import QtGui
-from Orange.data import Table
-from Orange.regression.random_forest import (RandomForestRegressionLearner,
-                                             RandomForestRegressor)
-from Orange.widgets.classify.owrandomforest import OWRandomForest
+from Orange.regression.random_forest import RandomForestRegressionLearner
+from Orange.widgets.classify import owrandomforest
 from Orange.widgets import settings
 
 
-class OWRandomForestRegression(OWRandomForest):
+class OWRandomForestRegression(owrandomforest.OWRandomForest):
     name = "Random Forest Regression"
     description = "Random forest regression algorithm."
+    icon = "icons/RandomForest.svg"
+    priority = 40
 
     LEARNER = RandomForestRegressionLearner
-    outputs = [("Learner", LEARNER),
-               ("Model", RandomForestRegressor)]
+    OUTPUT_MODEL_NAME = "Model"
```

Comment:
Having separate base classes could help us limit what connects to what (i.e. some widget accepts input 'Learner', but only of `OWBaseClassifier` type). if we ever decide for/need it.

How about simple overriding:
```py
class OWClsX(OWBaseClassifier):
    ...

class OWRegX(OWClsX):
    OUTPUT_MODEL_NAME = OWBaseRegressor.OUTPUT_MODEL_NAME
    ...
```
Seems the same number of symbols will need to be imported anyway. If you'd rather go the metaclass way, you're welcome too!